### PR TITLE
Update_FI-resources_v1

### PR DESCRIPTION
**Pull request by siirif**
more information links & relevant tools still missing. Working on them.

### CODE REVIEW

**Comment by siirif** [comment]
I cannot make the table for the resources & tools. Could someone help. Here some input data:
resources:
  - name: Chipster
    description: Chipster is a user-friendly analysis software for high-throughput data such as RNA-seq and single cell RNA-seq. It contains analysis tools and a large reference genome collection.
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, IT support]
      your_tasks: [data analysis]
    url: https://chipster.csc.fi/
  - name: DMPTuuli
    description: Data management planning tool (Finland)
    instance_of: DMPRoadmap
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, data manager]
      your_tasks: [DMP]
    url: https://www.dmptuuli.fi/
- name: Fairdata.fi
    description: With the Fairdata Services you can store, share and publish your research data with easy-to-use web tools.
    instance_of: 
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, data manager]
      your_tasks: [storage, data_publication, existing_data]
    url: https://research.csc.fi/-/fairdata-services
- name: Federated EGA Finland
    description: FEGA allows you to store and shaare sensitive data in Finland in a way that fulfils all the requirements of the General Data Protection Regulation (GDPR).
    instance_of: 
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, data manager]
      your_tasks: [sensitive_data, data_publication, existing_data]
      your_domain: [human_data]
      url: https://research.csc.fi/-/fega
- name: Findata
    description: The Health and Social Data Permit Authority. Findata offers services and enables secure and efficient utilisation of data materials containing health and social data.
    instance_of: 
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, data manager]
      your_tasks: [sensitive_data, existing_data]
      your_domain: [human_data]
      url: https://findata.fi/en/
- name: Fingenious
    description: Finnish Biobank Cooperative (FINBB) connects researchers to Finnish biomedical research. Via Fingenious® services the researcher can connect to all Finnish public bio banks.
    instance_of: 
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, data manager]
      your_tasks: [sensitive_data]
      your_domain: [human_data]
      url: https://site.fingenious.fi/en/
- name: Sensitive Data Services for Research
    description: CSC Sensitive Data Services for Research are designed to support secure sensitive data management through web-user interfaces accessible from the user's own computer
    instance_of: 
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, data manager]
      your_tasks: [sensitive_data, data_analysis, data_storage, data_publication]
      your_domain: [human_data]
      url: https://research.csc.fi/sensitive-data-services-for-research
- name: High performance computing
    description: CSC Supercomputers Puhti, Mahti and LUMI performance ranges from medium scale simulations to one of the most competitive supercomputers in the world.
    instance_of: 
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, data manager]
      your_tasks: [data_analysis]
      url: https://research.csc.fi/computing#high-performance-computing

- name: Cloud computing
    description: CSC offers a variety of cloud computing services: the Pouta IaaS services and the Rahti container cloud service.
    instance_of: 
    how_to_access: 
    related_pages:
      tool_assembly: [CSC]
      your_role: [researcher, data manager]
      your_tasks: [data_analysis]
      url: https://research.csc.fi/computing#cloud-computing

**Comment by bedroesb** [comment]
@siirif You head it almost right! YAML (the metadata) doesn't like colons, so you have to put text with a colon in quotes so it doesn't parse it as something else.

Other small problem I found was that the url was part of the related pages instead of the general object. 

We know this way of describing the resources is not the most easy one and prone to little mistakes, that we we help out!

**Comment by siirif** [comment]
I see thank you Bert! :) I'm slowly learning markdown 👍 

Can you fix those or should I still try? E.g. this one "CSC offers a variety of cloud computing services: the Pouta IaaS services and the Rahti container cloud service." could just be modified:  "CSC offers a variety of cloud computing service e.g. the Pouta IaaS services and the Rahti container cloud service."

This I didn't quite understand: "Other small problem I found was that the url was part of the related pages instead of the general object." Sorry and thank you so much for helping out 🥇

**Comment by bedroesb** [comment]
I already applied the obvious ones in my commit Update [fi_resources.md](https://github.com/elixir-europe/rdmkit/pull/816/commits/5246d0ab95135671e5dbeefe000181888ea3d6c9). Because of this the checks turned green :) But please have a double check I didn't change something wrong!

File: pages/national_resources/fi_resources.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Finland
 country_code: FI
 search_exclude: true
```
Comment (bedroesb) [inline pages/national_resources/fi_resources.md:4]:
```suggestion
```

If you are happy with the content this line can be deleted so the page appears in the navigation!

Comment (minnami) [reply pages/national_resources/fi_resources.md:4]: Looks good to me! Thanks @bedroesb

Comment (siirif) [reply pages/national_resources/fi_resources.md:4]: Yes thank you so much!

**Comment by lauportell** [comment]
closes #814 
Thank you Siiri for adding content and Bert for the review! I have checked the content and looks good to me - I will approve this pull request :) and if you don't need to add more content I will merge.

**Review by lauportell — APPROVED** [review]

**Comment by minnami** [comment]
Thanks @lauportell We don't have any more content to add at this point so please merge when convenient.

**Pull request merged**